src/Router.ts

```diff
@@ -1,221 +1,35 @@
-import Map from '@dojo/shim/Map';
-import { assign } from '@dojo/core/lang';
 import Task from '@dojo/core/async/Task';
-import Evented, { BaseEventedEvents } from '@dojo/core/Evented';
-import { Hash } from '@dojo/core/interfaces';
+import Evented from '@dojo/core/Evented';
+import { assign } from '@dojo/core/lang';
```

Comment:
ideally we would now import this from `@dojo/shim/object`

Reply:
good call.

Reply:
Updated